### ISSUE TITLE
Add ability to scroll advanced options

### ISSUE THREAD

**Issue by grenzor** [einaregilsson/Redirector#63]
Currently, it is hard if not impossible, especially when long links are in the redirect rule to check each box in Advanced Options and Save because once you zoom out the text becomes very very small and cannot click the save button.

Please add ability to scroll the advanced options until the save button.

**Comment by einaregilsson** [comment]
Not sure what you mean. This is what it looks like for me with a long value in the redirect boxes:

![screen shot 2016-06-20 at 10 07 12](https://cloud.githubusercontent.com/assets/557149/16190439/cbb52760-36ce-11e6-84e4-eaa56608d52b.png)

Can you post a screenshot of how it looks for you?

**Comment by grenzor** [comment]
At default 100% text/page zoom:  http://i.imgur.com/IzklCV0.png

For very short links it is possible to zoom out, check boxes and click save but when any of the 3 possible url boxes get long it is impossible to scroll down or zoom out and click save.

**Comment by einaregilsson** [comment]
But isn't that the scrollbar I see on the left of the image? Does that just not work?

**Comment by grenzor** [comment]
That scrollbar is for the page. It does not work for scrolling 'edit redirect' screen

**Comment by einaregilsson** [comment]
Ok, will try to get this fixed in the next version. Not sure when that will be though.

**Comment by laniakea64** [comment]
> Ok, will try to get this fixed in the next version. Not sure when that will be though.

Hi @einaregilsson , any update on this?  In Redirector 3.5.1 the "Save" and "Cancel" buttons are completely inaccessible below the window viewport for me when "Show advanced options" is used, even for a redirect example result that fits on one line.

**Comment by Crystal-RainSlide** [comment]
My solotion:
``` CSS
#edit-redirect-form {
	max-height: 80vh;
	overflow: auto;
}
```

**Comment by einaregilsson** [comment]
Just published 3.5.2 which adds scrolling for the form.
